### ISSUE TITLE
An Information Builder Component for Provider Description

### ISSUE THREAD

**Issue by jakezam** [jakezam/crabapple-conglomerate#23]
We want to gather specifics about provider to build their information component on their profile

Information we want:

- How long have they been doing this
- What their experience has been
- What are their pricing options
- What are their general strengths
- What projects in their field are they most comfortable with/have completed
- What are their target locations (How far are they willing to go)
- A text field for them to put in additional information

This is just what I came up with, if either you believe a point is unnecessary or I am missing a point feel free to make any necessary changes

**Comment by 0NotApplicable0** [comment]
On provider account creation this is implemented, do we want to also add an external option on the settings page?

**Comment by jakezam** [comment]
Yes, we want to give people the ability to modify/add to this stuff, which can either be in settings or right on the profile page
